pact.cabal

```diff
@@ -90,6 +90,7 @@ library
                      , Pact.Typechecker
 
   if flag(ghc-flags)
+    build-tool-depends: hsinspect:hsinspect
```

Comment:
What is this? Will it impact devs if they don't have this?

Reply:
This is just for my emacs setup, which suddenly switched to require this (where it used to be globally managed). It's hidden behind the flag.